### PR TITLE
💀 Skeleton UI & resource name fixes

### PR DESCRIPTION
**Pull request by dangodev**
## Reason for change
Adds loading states to the marketplace, product, and plan components

skeleton

skeleton-product

skeleton-plan

### CODE REVIEW

**Comment by davidleger95** [comment]
Fancy! ✨ ✨ ✨

File: src/data/marketplace.ts [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+      ...body,
+      name: 'James Bowes',
+      tagline:
+        'James Bowes: the mighty moustache, paladin of all things technical and distrubuted.',
+      tags: ['database', 'logging', 'messaging', 'optimization', 'runtime', 'sell-your-service'],
+    },
+    type: 'product',
+    version: 1,
+  },
+];
```
Comment (davidleger95) [inline src/data/marketplace.ts]: lol, nice sample text 😂

File: src/components/manifold-active-plan/manifold-active-plan.tsx [hunk 2]
```diff
@@ -15,20 +15,18 @@ export class ManifoldActivePlan {
   @Prop() regions?: string[];
   @Prop() selectedResource?: Gateway.Resource;
   @State() selectedPlanId: string;
+  @Watch('plans') plansChange(newPlans: Catalog.ExpandedPlan[]) {
+    if (this.selectedResource && this.selectedResource.plan && this.selectedResource.plan.id) {
+      this.selectPlan(this.selectedResource.plan.id);
+    }
+    this.selectPlan(newPlans[0].id);
```
Comment (davidleger95) [inline src/components/manifold-active-plan/manifold-active-plan.tsx]: do you want to do this even if the if-condition is true?

Comment (dangodev) [reply src/components/manifold-active-plan/manifold-active-plan.tsx]: Good catch!

**Pull request merged**